### PR TITLE
Revert back eslint-plugin-wix-style-react version check

### PR DESCRIPTION
**Pull request by yavorsky**
This reverts commit 6a44e94a1d7384be8ff2678cf633316dbee8fb44.

### CODE REVIEW

File: packages/eslint-plugin-wix-style-react/tests/fixtures/disable-no-full-wsr-lib/no-full-wsr-lib.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    sourceType: 'module'
+  }
+});
+
+//------------------------------------------------------------------------------
+// Tests
+//------------------------------------------------------------------------------
+const ruleTester = new RuleTester();
+ruleTester.run('no-full-wsr-lib', rule, {
+  valid: [`import {Button} from 'wix-style-react';`],
```
Comment (argshook) [inline packages/eslint-plugin-wix-style-react/tests/fixtures/disable-no-full-wsr-lib/no-full-wsr-lib.js]: @yavorsky 

this case is failing, just `npm test` it. that's why i reverted, lot's of builds were red, complaints incoming.

do i miss something?

Comment (yavorsky) [reply packages/eslint-plugin-wix-style-react/tests/fixtures/disable-no-full-wsr-lib/no-full-wsr-lib.js]: @argshook `npm test` for this project or for the whole monorepo in general ?

Comment (argshook) [reply packages/eslint-plugin-wix-style-react/tests/fixtures/disable-no-full-wsr-lib/no-full-wsr-lib.js]: `npm test` for this project. do you write tests without running them? :)

Comment (yavorsky) [reply packages/eslint-plugin-wix-style-react/tests/fixtures/disable-no-full-wsr-lib/no-full-wsr-lib.js]: screen shot 2019-01-15 at 3 51 19 pm

Comment (yavorsky) [reply packages/eslint-plugin-wix-style-react/tests/fixtures/disable-no-full-wsr-lib/no-full-wsr-lib.js]: Seems like `mocha './tests/{,!(fixtures)/**}/*.js' --recursive` acting differently on different OS/envs and `fixtures` are not being ignored for CI.

**Pull request merged**